spdx-utils/build.gradle

```diff
@@ -86,16 +86,55 @@ task generateSpdxLicenseList {
 
         def enumValues = spdxLicenseIds.collect { "    ${licenseToEnumEntry(it.toString())}" }.sort().join(',\n') + ';'
         licenseEnumFile.append(enumValues)
-        licenseEnumFile.append("""
+        licenseEnumFile.append('''
 
                 companion object {
                     /**
                      * Return the enum value for the given [id], or null if it is no SPDX license id.
                      */
                     fun forId(id: String) = SpdxLicense.values().find { it.id == id }
                 }
+
+                /**
+                 * The full license text as a string.
+                 */
+                val text = javaClass.getResource("/licenses/$id").readText()
             }
-            """.stripIndent())
+            '''.stripIndent())
+
+        println 'Fetching SPDX license texts...'
+        def count = 0
+
+        spdxLicenseIds.each { id ->
+            def licenseTextResourceFile = file("src/main/resources/licenses/$id")
```

Comment:
Should we already use a subdirectory "licenses/spdx" because we might also add license texts for LicenseRefs later?

Reply:
Good question. In a way, LicenseRefs to me also *are* SPDX ids, just not "core" / "built-in" ones. And given that ids for LicenseRefs will always start with `LicenseRef-` (also regarding the file name) I though the separation is clear enough. But I'm not totally against your idea, just a bit undecided. What directory name would you propose for LicenseRefs then?

Reply:
At least in the case of ScanCode per-scanner LicenseRef sub-directories would not really be needed as [ScanCode has recently started](https://github.com/nexB/scancode-toolkit/pull/1307) to use a `LicenseRef-scancode-` prefix.

Reply:
Ok, let's keep it as is, we can change it later if required.